src/pages/AdminCallbackPage/AdminCallbackPage.tsx

```diff
@@ -0,0 +1,23 @@
+import { useState, useMemo } from "react";
+import { v4 as uuidv4 } from "uuid";
+import { useInitialisedDeskproAppClient } from "@deskpro/app-sdk";
+import { AdminCallback } from "../../components";
+import type { FC } from "react";
+import type { Maybe } from "../../types";
+
+const AdminCallbackPage: FC = () => {
+  const [callbackUrl, setCallbackUrl] = useState<Maybe<string>>(null);
```

Comment:
To reduce type juggling, instead of `useState<Maybe<string>>(null)`, I think it should be `useState<string|undefined>(undefined)`